docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '2.1'
```

Comment:
Docker-compose in stretch is version 1.8, and seems to only support up to version 2.0 of the config file.  Downgrading this version string fixes the problem for me, so I'll add a follow-up patch.